satellite/analytics/service.go

```diff
@@ -132,6 +232,60 @@ func (service *Service) TrackCreateUser(fields TrackCreateUserFields) {
 		Event:       eventAccountCreated,
 		Properties:  props,
 	})
+
+	fullName := fields.FullName
+	names := strings.SplitN(fullName, " ", 2)
+
+	var firstName string
+	var lastName string
+
+	if len(names) > 1 {
+		firstName = names[0]
+		lastName = names[1]
+	} else {
+		firstName = fullName
+	}
+
+	data := payloadUser{
+		Email: fields.Email,
+		Properties: propertiesUser{
+			Email:          fields.Email,
+			Firstname:      firstName,
+			Lastname:       lastName,
+			Lifecyclestage: "customer",
+		},
+	}
+
+	dataEvent := map[string]interface{}{
+		"email":     fields.Email,
+		"eventName": "pe20293085_account_created_new",
+		"properties": map[string]interface{}{
+			"userid":             fields.ID.String(),
+			"email":              fields.Email,
+			"name":               fields.FullName,
+			"satellite_selected": service.satelliteName,
+			"account_type":       string(fields.Type),
+			"company_size":       fields.EmployeeCount,
+			"company_name":       fields.CompanyName,
+			"job_title":          fields.JobTitle,
+			"have_sales_contact": fields.HaveSalesContact,
+		},
+	}
+
+	go func() {
```

Comment:
Do not use naked `go` in code, this usually means there's a subtle bug somewhere.